pages/index.tsx

```diff
@@ -11,7 +13,7 @@ export default function Home() {
       <Head>
         <title>First Issue | Issues for your next open-source contribution</title>
       </Head>
-      <RepositoryList repositories={repositories} />
+      <RepositoryList repositories={repositories} dtOrder={data.date} starOrder={data.star}/>
```

Comment:
Since the sorting strategy will be global, it is something we should put in the React Context rather than on a file